fix(debPacker): allow absolute paths in copy-files and mkdirs (#4585)
ovh/cds: tools/debpacker/README.md

# DebPacker

> Creates a debian package

Reviewer: ```suggestion
Creates a Debian package and a systemd configuration
```

## prerequisite

* You must have a sane instance of `go`
* You must be in a debian environment (needs `dpkg`)

## Usage

> The programme needs a configuratoin file to work. This configuration file describes how to build the package

Reviewer 2: ```suggestion
The program needs a configuration file to work. This configuration file describes how to build the package
```

Email of the maintener

### dependencies

**Type:** *string[]*

List of dependancies

### systemd-configuration

Configure service

#### after

**Type:** *string*

After

#### user

**Type:** *string*

User that launch the binary

#### args

**Type:** *string[]*

Arguments to pass to the binary

#### stop-command

**Type:** *string*

Stop command

#### restart

**Type:** *string*

Restart policy. For instance `always`

#### wanted-by

**Type:** *string*

Wanted by

#### environments

**Type:** *dictionary*

Environment variables

#### working-directory

**Type:** *string*

Directory in which the binary is launched

#### post-install-cmd

**Type:** *string*

Post install command

#### auto-start

**Type:** *boolean*

If true, start the service during package installation

## Examples

### Executable

```yaml
package-name: "my-executable"
architecture: "all"
binary-file: "./my-exec"
copy-files:
  - "./data/*.txt:data"
version: "0.0.1"
description: "Executable example"
maintainer: "mickey@mouse.com"
systemd-configuration:
  user: "root"
  after: network.target
  stop-command: /bin/kill $MAINPID
  restart: always
  wanted-by: multi-user.target
  working-directory: "/var/lib/foo"
```

> ### hml client

Reviewer: ```suggestion
### html client
```
